src/device/working_cpu/eBPF_Core/.Xil/Vivado-23328-DESKTOP-TU0BN25/realtime/CPU.tcl

```diff
@@ -1,139 +0,0 @@
-# 
```

Comment:
Hi Martin,

I'm having some difficulty combining the commits, it's not something I'm familiar with doing, can you provide any info on how to get these merged into two commits instead of how it currently is?

Reply:
Hi Andrew,

What you're looking for here is to squash the two commits. See for example this tutorial: https://www.internalpointers.com/post/squash-commits-into-one-git

Note that if you need to split two commits, you can do it like this: https://emmanuelbernard.com/blog/2014/04/14/split-a-commit-in-two-with-git/